### DISCUSSION TITLE
MailerSend integration

### DISCUSSION THREAD

**Discussion by Agathe-Brusset** [supabase/supabase#9533] [Feature Requests]
# Feature request
Ability to send notifications that are not auth-related, like comments on posts, invoices and other event-triggered communications. 

## Is your feature request related to a problem? Please describe.
Users are unable to send emails that are not SMTP or not authentication related. They are limited in the types of email they can send and they could have an integration in place to have better control over the email contents, and have them match their brand identity.

## Describe the solution you'd like
Integration with a 3rd party tool like MailerSend, to offer transactional communications (Email and SMS) API to have more design options and trigger them on different events. 

## Describe alternatives you've considered
Cf discussion [here](https://github.com/supabase/supabase/discussions/1194).

## Additional context
I'm Agathe Brusset, MailerSend's product manager. MailerSend is a transactional communications API, we're aimed to provide high email deliverability and an easy user experience for the whole team. Our API reference is [here](https://developers.mailersend.com/) and we can provide a test account to create the integration.

**Comment by christianfl** [comment]
Already using MailerSend within Supabase via https://github.com/burggraf/supabase-mailer This works like a charm.

**Comment by kiwicopple** [comment]
In the future we will bring out a workflow services which should cover this use case. We don't have a timeline on it, but it's in the backlog.

For now, @christianfl 's link is good, or you can use our Edge Functions to send mail (even trigger them with Database Webhooks and pgcron).

I'll convert this to a Discussion, because it's useful for gathering ideas. Thanks for the suggestion @Agathe-Brusset !
